### PR TITLE
Issue 36891

### PR DESCRIPTION
**Pull request by labkey-bpatel**

### CODE REVIEW

File: api/src/org/labkey/api/query/DefaultQueryUpdateService.java [hunk 1]
```diff
@@ -381,6 +413,54 @@ public void updateStatistics(int currentRow)
         return row;
     }
 
+    protected void validateValue(ColumnInfo column, Object value) throws ValidationException
+    {
+        DomainProperty dp = getDomain() == null ? null : getDomain().getPropertyByName(column.getColumnName());
+        List<ColumnValidator> validators = ColumnValidators.create(column, dp);
+        for (ColumnValidator v : validators)
+        {
+            String msg = v.validate(1, value, _validatorContext);
```
Comment (labkey-jeckels) [inline api/src/org/labkey/api/query/DefaultQueryUpdateService.java]: It seems like this rowNum value is hardly used, but suggest passing in -1 or some other obviously wrong value if we aren't keeping track of the real number

File: api/src/org/labkey/api/query/UserSchemaAction.java [hunk 2]
```diff
@@ -281,14 +281,10 @@ else if (table.getPkColumnNames().size() > 1)
                     throw x;
                 errors.reject(SpringActionController.ERROR_MSG, x.getMessage());
             }
-            catch (InvalidKeyException | DuplicateKeyException | DataIntegrityViolationException | RuntimeSQLException x)
+            catch (InvalidKeyException | DuplicateKeyException | DataIntegrityViolationException | RuntimeSQLException | BatchValidationException x)
             {
                 errors.reject(SpringActionController.ERROR_MSG, x.getMessage());
             }
-            catch (BatchValidationException x)
```
Comment (labkey-jeckels) [inline api/src/org/labkey/api/query/UserSchemaAction.java]: Why did this change? BatchValidationExceptions can have multiple errors embedded in them, and this changed approach loses any additional ones

File: issues/src/org/labkey/issue/query/IssuesListDefTable.java [hunk 3]
```diff
@@ -76,6 +76,13 @@
 {
     private static final Logger LOG = Logger.getLogger(IssuesListDefTable.class);
 
+    private final static Set<String> _AUTOPOPULATED_COLUMN_NAMES;
+    static
+    {
+        _AUTOPOPULATED_COLUMN_NAMES = new HashSet<>(Table.AUTOPOPULATED_COLUMN_NAMES);
+        _AUTOPOPULATED_COLUMN_NAMES.add("Name");
```
Comment (labkey-jeckels) [inline issues/src/org/labkey/issue/query/IssuesListDefTable.java]: Please make this an immutable set

File: api/src/org/labkey/api/query/DefaultQueryUpdateService.java (hunk 1 above)
Comment (labkey-bpatel) [reply api/src/org/labkey/api/query/DefaultQueryUpdateService.java]: done!

File: api/src/org/labkey/api/query/UserSchemaAction.java (hunk 2 above)
Comment (labkey-bpatel) [reply api/src/org/labkey/api/query/UserSchemaAction.java]: agreed. I've added that back.

File: issues/src/org/labkey/issue/query/IssuesListDefTable.java (hunk 3 above)
Comment (labkey-bpatel) [reply issues/src/org/labkey/issue/query/IssuesListDefTable.java]: Done! let me know if there's a better way, my change looks a bit clunky.

**Pull request merged**